### PR TITLE
Dont exit early on OPTIONS requests

### PR DESCRIPTION
**Pull request by roborourke**
OPTIONS requests should be able to return the available schema / routes for a given endpoint in the body. This was causing problems with pre-flight checks for logged in users because of the early exit in the code.

Fixes https://github.com/humanmade/altis-security/issues/76

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -26,7 +26,6 @@ function redirect_user() {
 			header( 'Access-Control-Allow-Credentials: true' );
 			header( 'Access-Control-Allow-Headers: Authorization, Content-Type' );
 			header( 'Access-Control-Allow-Expose-Headers: X-WP-Total, X-WP-TotalPages' );
-			exit;
```
Comment (rmccue) [inline inc/namespace.php:29]: I don't think this is enough, as it'll fail the `$page` check below. Also, if we're sending the full OPTIONS response, we don't need to send these manual headers

**Review by rmccue — CHANGES_REQUESTED** [review]

File: inc/namespace.php (hunk 1 above)
Comment (joehoyle) [reply inc/namespace.php:29]: Yeah I'm not sure that will do what is intended. Can we just live this block below the early returns on logged in / allowed pages checks?

Comment (rmccue) [reply inc/namespace.php:29]: Yeah actually, I think this whole block just wants to live down after the cap check, and that'll sort it

Comment (roborourke) [reply inc/namespace.php:29]: It's working in the sense I can add audiences again but yeah makes sense - I'll put this in the block after the logged in check that handles REST requests

Comment (joehoyle) [reply inc/namespace.php:29]: Yeee that's what made me worry about putting specific success criteria :D

**Review by rmccue — APPROVED** [review]

**Pull request merged**